projects/core/observables/index.ts

```diff
@@ -1,2 +1,2 @@
 export * from './smart-search';
-export * from './zoneful-map';
+export * from './zonefull-map';
```

Comment:
Should we also revert filename ?

https://github.com/Tinkoff/taiga-ui/pull/4292#discussion_r1175277003

Reply:
oh, sorry